src/generators/java/presets/ConstraintsPreset.ts

```diff
@@ -7,9 +7,12 @@ import { CommonModel } from '../../../models';
  * 
  * @implements {JavaPreset}
  */
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 export const JAVA_CONSTRAINTS_PRESET: JavaPreset = {
   class: {
+    self({renderer, content}) {
+      renderer.addUniqueDependency('import javax.validation.constraints.*;');
+      return content;
```

Comment:
I think that we can change the line https://github.com/asyncapi/modelina/blob/master/src/generators/AbstractRenderer.ts#L50 and return (regarding to my last comment in PR) the latest not falsey value of content, so we can change the mentioned line to:

```ts
        content = await preset[String(functionName)]({ 
            ...params,           
            renderer: this,           
            content,           
            options,           
            model: this.model,           
            inputModel: this.inputModel        
        }) || content;
```

and then you don't need to write `return content;` in every preset if you add only deps :)

Reply:
While I would do that, it requires further refactoring and testing, which I do not want to introduce in this PR 🙂 

Good suggestion though, maybe we should add a new issue?

Reply:
Ok, make it as followup :)

Reply:
@magicmatatjahu are you gonna add the issue or should I?

Reply:
Created https://github.com/asyncapi/modelina/issues/247